compact/far_backward/definitions.xml

```diff
@@ -1,18 +1,16 @@
 <!-- SPDX-License-Identifier: LGPL-3.0-or-later -->
 <!-- Copyright (C) 2022 Simon Gardner, Jaroslav Adam, Wouter Deconinck -->
 
-<lccdd>
```

Comment:
You have to stay in the lccdd namespace. Why did you remove this?

Reply:
It was removed to mirror the situation in the main and far_forward definitions.xml files. It might be nicer and more consistent to remove the surrounding define block from the top xml file and add lccdd back into these, allowing flexibility to nest the define files.

Reply:
do the content of define blocks take precedence over the include blocks file content? Moving the main definitions file out of the define and adding lccdd tags to it means world_dz is not yet defined when the world block is processed.

Reply:
Ok, let's leave as is.